### DISCUSSION TITLE
Client Integration

### DISCUSSION THREAD

**Discussion by AzraAnimating** [tidal-music/discussions#9] [Ideas & Feature requests]
It would be great to be able to integrate the Tidal client into some Software of our own, something like whats possible with Spotify's API, where you can control Playback and Queue options.

**Comment by stefanmoro** [comment]
Hi @AzraAnimating and thank you for joining TIDAL community discussions!

If I understand your question correctly, you are referring to the functionality that Spotify supports on e.g. iOS and Android, where you can build an app and interact with the Spotify app running in the background?

Unfortunately we are currently not planning for such a solution. However, we hope that with time, the TIDAL Developer Platform will allow developers to build new experiences that include playback of media through the TIDAL ecosystem.

**Reply by alexandru0-dev** [reply]
First of all, love the fact that Tidal created a Developer Program and thanks to you and all the people behind it.

Is Tidal Connect going to be publicly documented?
Probably with that u can find a way to do remote control.
I would love to see this public as even if Tidal Connect exists, i cannot remote control the desktop app thru my phone and vice versa

**Reply by stefanmoro** [reply]
Hey @alexandru0-dev, and welcome to the forum.

As of today, TIDAL connect integrations goes through a separate process. Please see https://tidal.com/connect for further information.

**Comment by ungive** [comment]
I would also love to see something like playback control, where we get the ability to play/pause a selected track, given a TIDAL track resource ID. For my case, this would allow something like "listening along" with someone else, where one person can control what is played and the other person (or more than one person) listens along with them.
